docs/project-configuration/overview.mdx

```diff
@@ -30,10 +31,20 @@ import Icon from "@site/static/icons/dataflow-01.svg";
         API, as well as collaborate with your team.
       </p>
       <p>
-        In this section you can learn about how metadata is <a href={"/project-configuration/config"}>configured</a>,
-        how to define <a href={"/project-configuration/projects"}>projects</a> with{" "}
-        <a href={"/project-configuration/subgraphs"}>subgraphs</a>, and how to bring it all together to create{" "}
-        <a href={"/project-configuration/builds"}>builds</a>.
+        You can use the Hasura <Link to="/cli/overview">DDN CLI</Link> to scaffold, author, validate and manage your
+        metadata, and the Hasura{" "}
+        <Link to="https://marketplace.visualstudio.com/items?itemName=HasuraHQ.hasura">VS Code extension</Link> to help
+        you further generate, auto-complete and validate your metadata. (Check out the{" "}
+        <Link to="/getting-started/quickstart">Quickstart</Link> if you haven't already to get a supergraph up and
+        running.)
```

Comment:
Do you think this approach is discoverable enough?

Would it make sense to give this more visibility on the `Overview`, or perhaps consider another way to organize it for clarity?  It might require some repetition across the different pages in this directory, but if we added this in the `Introduction` of each page, it could aid users in understanding they need to have scaffolded a project using the CLI.